python/CMakeLists.txt

```diff
@@ -56,6 +56,9 @@ if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
   option(PYARROW_BUILD_JEMALLOC
     "Build the PyArrow jemalloc integration"
     OFF)
+  option(PYARROW_BUNDLE_ARROW_CPP
+    "Bundle the Arrow C++ libraries" 
+    OFF)
```

Comment:
I will have to refactor this a bit but it won't change much sadly. Packaging binary libs in a manylinux1 wheel is cumbersome.